elliott/elliottlib/cli/create_textonly_cli.py

```diff
@@ -108,10 +108,6 @@ def create_textonly(runtime, errata_type, date, assigned_to, manager, package_ow
         erratum.commit()
         green_prefix("Created new text only advisory: ")
         click.echo(str(erratum))
-        bug = bug_tracker.create_textonly(bug_title, bug_description)
```

Comment:
This line shouldn't be removed.

Reply:
text-only advisory is also RHOSE product which doesn't need a placeholder bug

Reply:
ack